monai/bundle/properties.py

```diff
@@ -58,16 +58,17 @@
         BundleProperty.REQUIRED: True,
         BundlePropertyConfig.ID: f"train{ID_SEP_KEY}dataset",
     },
-    "train_dataset_data": {
-        BundleProperty.DESC: "data source for the training dataset.",
-        BundleProperty.REQUIRED: True,
-        BundlePropertyConfig.ID: f"train{ID_SEP_KEY}dataset{ID_SEP_KEY}data",
-    },
     "train_inferer": {
         BundleProperty.DESC: "MONAI Inferer object to execute the model computation in training.",
         BundleProperty.REQUIRED: True,
         BundlePropertyConfig.ID: f"train{ID_SEP_KEY}inferer",
     },
+    "train_dataset_data": {
```

Comment:
Hi @SachidanandAlle , could you please help to review this change? Thanks in advance!